frontend/src/App.tsx

```diff
@@ -13,3 +13,31 @@ function App() {
 }
 
 export default App;
+
+/*
+import {ethers} from 'ethers';
+import deploy from './deploy';
+import displayContract from './displayContract';
+import "./index.scss";
+
+let contracts = 0;
+async function newContract() {
```

Comment:
wait, we are deploying contract right from the UI? do we really need this functionality right now? or should we prioritize end user journey first?

Reply:
actually this code shouldn't even be in this PR. apologies. juggling code migration and branches from old and new laptop. but to your point, yep, i agree. i had started this a bit earlier before our call last night.

not all is lost, as i needed to read up on react components and react-hook-forms (albeit a single input form of 'deposit amount'). let me remove this file from the PR, and we won't let just anyone create contracts yet. it will all be focused on the depositor for now.